blog/2023-01-10-first-offsite/index.mdx

```diff
@@ -0,0 +1,123 @@
+---
+slug: first-offsite-retro
+title: Lessons from Firezone’s first offsite
+authors: [jason]
+tags: [company, offsite]
+---
+
+My first startup job was at [Strong](https://www.strong.app/), a workout journal app that was pretty popular at the time. The founders were “digital nomads,” and the team was fully remote.
+
+Back in 2016, this was highly uncommon.
+
+We spent a lot of time creating processes to work together effectively on opposite sides of the world. One of these rituals was regularly meeting up in person.
+
+![digital nomads](https://user-images.githubusercontent.com/52545545/211626078-6f64a4f4-ec37-40cd-8bb5-a6b9b122f351.png)
+
+At first, it was by accident. Since everyone was traveling anyway, we would coordinate being in the same city at the same time.
+
+After working in person we quickly noticed, the way we worked remotely improved. People were more comfortable sharing their ideas, better at giving constructive criticism, and happier in general.
+
+It felt better to work together.
+
+## Why try this at Firezone?
+
+If you believe a small network security startup can take a sizable bite out of the usual suspects of legacy VPN products, there are a few ways it can happen. Many involve assembling an all-star team who can execute quickly to build a great product.
```

Comment:
```suggestion
If you believe a small network security startup can take a sizable bite out of the usual suspects of legacy VPN products, there are a few ways it can happen. Many involve assembling an all-star team who can execute quickly to build a great product.
```

I'm not sure I follow this paragraph -- the following two paragraphs don't seem to follow smoothly from it?